packages/installer.vm/tools/chocolateyinstall.ps1

```diff
@@ -0,0 +1,222 @@
+$ErrorActionPreference = 'Continue'
+$global:VerbosePreference = "SilentlyContinue"
+Import-Module vm.common -Force -DisableNameChecking
+
+try {
+    # Gather packages to install
+    $installedPackages = (VM-Get-InstalledPackages).Name
+    $configPath = Join-Path ${Env:VM_COMMON_DIR} "packages.xml" -Resolve
+    $configXml = [xml](Get-Content $configPath)
+    $packagesToInstall = $configXml.config.packages.package.name | Where-Object { $installedPackages -notcontains $_ }
+
+    # List packages to install
+    Write-Host "[+] Packages to install:"
+    foreach ($package in $packagesToInstall) {
+        Write-Host "`t[+] $package"
+    }
+    Start-Sleep 1
+
+    # Install the packages
+    try {
+        foreach ($package in $packagesToInstall) {
+            Write-Host "[+] Installing: $package" -ForegroundColor Cyan
+            choco install "$package" -y
+            VM-Write-Log "INFO" "Package $package has been installed"
+        }
+    } catch {
+        VM-Write-Log-Exception $_
+    }
+    VM-Write-Log "INFO" "[+] All packages complete"
+
+    # Set Profile/Version specific configurations
+    VM-Write-Log "INFO" "[+] Beginning Windows OS VM profile configuration changes"
+    VM-Apply-Configurations $(Join-Path $Env:VM_COMMON_DIR "config.xml")
+
+    # Configure PowerShell and cmd prompts
+    VM-Configure-Prompts
+
+    # Configure PowerShell Logging
+    VM-Configure-PS-Logging
+
+    # Configure Desktop\Tools folder with a Mandiant VM icon
+    $folderPath = $Env:TOOL_LIST_DIR  
+    $iconPath = Join-Path $Env:VM_COMMON_DIR "vm.ico"
```

Comment:
```suggestion
    $iconPath = Join-Path $Env:VM_COMMON_DIR "vm.ico" -Resolve
```

Is there a case this icon file wouldn't exist? For instance, do we have a default if the use doesn't select one?

Reply:
I added the resolve to ensure the file exists before using later

Reply:
There shouldnt be a case, so yes lets stick with `-Resolve`.. I had the commando vm icon just converted to an ico and then it gets saved in the commando install script:
```
# Set ico file
$icoImage = "${Env:VM_COMMON_DIR}\vm.ico"
$sourceIco = Join-Path $PSScriptRoot "Images\commando.ico"
Copy-Item -Path $sourceIco -Destination $icoImage
```

Reply:
Gotcha, just remember that the "-Resolve" flag will throw an error if a path doesn't exist, so you may need a try catch if you use it so the script can still continue versus bailing too early

Reply:
Otherwise, you can manually test the path